test/unit/context/context_mm_black.h

```diff
@@ -46,11 +46,15 @@ class ContextBlack : public CxxTest::TestSuite {
     unsigned chunkSizeBytes = 16384;
     unsigned maxFreeChunks = 100;
     unsigned piecesPerChunk = 13;
-    unsigned len = chunkSizeBytes / piecesPerChunk; // Length of the individual block
-    unsigned N = maxFreeChunks*piecesPerChunk;
+    unsigned len;
+    unsigned N;
+
+    len = chunkSizeBytes / piecesPerChunk;  // Length of the individual block
+    N = maxFreeChunks * piecesPerChunk;
```

Comment:
Why move the variable initializations? [I prefer to always initialize variables when declaring them.](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Res-always)

Reply:
Because `len` is reused and that's why *I* prefer to have it declared without assignment (if it is a simple type), because it is easier to see it that way.